test/JDBC/input/BABEL_4553.mix

```diff
@@ -0,0 +1,102 @@
+-- tsql
+CREATE DATABASE dms_test_target_db
+go
+USE dms_test_target_db
```

Comment:
Also, let us have some perf testing. IMO, we should get a bit better perf numbers as we have modified the join condition to check the oid only whereas initially there were around 4 operations involved. Anyways, let us have perf related tests as well.